Update README.md

kiryteo opened the pull request.

oshadura commented:
@kiryteo can you please squash all commits together (they all have the same title)?

oshadura commented on README.md:
```diff
@@ -1,2 +1,203 @@
+
+- ROOT's package manager prototype
+
+root-get is the project centric package manager for ROOT. The application provides simplified build and install process for ROOT packages and modules. 
```
Can we mention here also that we work with external packages too?

kiryteo replied:
ok, I have explained it later but I'll add here too.

oshadura commented on README.md:
```diff
@@ -1,2 +1,203 @@
+
+The directory path where user can get the installed package/module.
+
+- ROOTSYS
```
Not exactly, ROOTSYS is also the directory where you unpacked the ROOT (e.g. ROOT  base)

kiryteo replied:
Here, Line 31 is the explanation for ROOT_PKG_PATH, for ROOTSYS the explanation is below it.

kiryteo commented:
Yes, sorry for the mess, I'll squash the commits.

oshadura commented on README.md:
````diff
@@ -1,2 +1,203 @@
+  deps: 'dependencies'
+```
+
+- Example module.yml :
````
Lets use here libhello?

kiryteo replied:
yes, actually it would be good thing to do.! thanks.!

oshadura commented on README.md:
```diff
@@ -1,2 +1,203 @@
+
+python3
+
+[matplotlib](https://matplotlib.org/users/installing.html)
```
what is a command that user will need to use to install matplotlib and networkx?

kiryteo replied:
The links are directed to official installation instructions only.

oshadura commented on README.md:
```diff
@@ -1,2 +1,203 @@
+
+
+## Initial setup
+Required - root build files directory, if not present already, do an out of source build for root
```
Lets remove installation of ROOT base package from here (it is not really belongs to steps how to setup root-get)?

oshadura replied:
I will move it in docs/setup-ROOTbase.md

kiryteo replied:
Sure, I'll create a separate file setup-ROOTbase.md and add there

oshadura commented on README.md:
```diff
@@ -1,2 +1,203 @@
+
+## Installing
+
+To install a new module with root-get :
```
Can we add here all other options we have? "-mi", "-pi", "-s", "-l" and etc.

kiryteo replied:
Yes, I'll do it. with subheadings for each of these.

oshadura commented on README.md:
````diff
@@ -1,2 +1,203 @@
+```
+$ mkdir installdir
+```
+Inside the root-get directory, set environment variables
````
why only inside root-get directory?:)

kiryteo replied:
Need to remove this.. its not necessary.

kiryteo committed:
Multiple changes
- Added example module manifest - self hosted repository, no third party usage
- Added commands with respect to recent changes

kiryteo commented:
I have added the discussed modifications and updated the readme.

kiryteo commented:
@oshadura  I have updated the README, please have a look and suggest if any changes are required..
